src/Host/Client/Test/Session/RSessionTest.InteractionEvaluation.cs

```diff
@@ -79,6 +79,42 @@ public class InteractionEvaluation : IAsyncLifetime {
                     completedTasks.Single().Result.Dispose();
                 }
             }
+
+            [Test]
+            [Category.R.Session]
+            public async Task NestedInteraction() {
```

Comment:
Since this test should check if RHost isn't crashing, you may want to watch for RHost task:
```{csharp}
_testMethodFixture.ObserveTaskFailure(_session.RHost.GetRHostRunTask());
```